app/views/hiring_staff/vacancies/review_vacancy_sections/_review_supporting_documents.html.haml

```diff
@@ -0,0 +1,16 @@
+%h2.govuk-heading-m.mb0
+  = t('jobs.supporting_documents')
+
+.govuk-body-s.mb1
+-#   = link_to documents_school_job_path, class: 'govuk-link', 'aria-label': 'Change supporting documents' do
```

Comment:
NB: This will be added and tested as part of [UD8](https://dfedigital.atlassian.net/browse/TEVA-414) instead.